_posts/2022-07-24-opensearch-plugin-zips-now-in-maven-repo.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "OpenSearch plugin zips now in Maven repo"
+authors: 
+  - prudhvigodithi
+date: 2022-07-24 01:01:01 -0700
+categories: 
+  - technical-post
+twittercard:
+  description: "Details on how to consume OpenSearch plugin zips from maven repo and the process involved in shipping them to the maven repo."
```

Comment:
Capitalize both instances of "Maven". Additionally, should it be "Details on how to consume OpenSearch plugin zips from a Maven repo and the process involved in shipping them to a Maven repo."?